x-pack/plugins/observability_solution/apm/server/plugin.ts

```diff
@@ -219,6 +224,67 @@ export class APMPlugin
       })
     );
 
+    plugins.observability.alertDetailsContextService.registerHandler(
```

Comment:
Can you extract the handler so in the plugin.ts file we only have `plugins.observability.alertDetailsContextService.registerHandler(apmAlertDetailsContextHandler)` ?